README.md

```diff
@@ -57,24 +57,36 @@ After you have installed GOV.UK Frontend the `@govuk-frontend` package will appe
 
 ## Import assets
 
-Import styles into your main Sass file.
+### Import styles
 
-To import all components, add the line:
+You need to import the GOV.UK Frontend styles into the main Sass file in your project.
+
+1. To import all components, add the below to your Sass file:
 ```CSS
 @import "@govuk-frontend/all/all";
 ```
 
-To import an individual component (for example, a button), add the line:
+2. To import an individual component (for example a button), add the below to your Sass file:
 ```CSS
 @import "@govuk-frontend/button/button";
 ```
 
+You should place the above code before your other Sass rules, or Sass imports, if you want to override
+GOV.UK Frontend with your own styles.
+
+In order to resolve the above `@import` paths(s) in your build, you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) (in Ruby, they should be added to [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths)).
 
-To resolve your `@import` declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) in Ruby.
+For example, if your project uses Gulp, you would add the Sass include paths to your Gulp configuration file (for example `gulpfile.js`) with [gulp-sass](https://www.npmjs.com/package/gulp-sass). Below is an example:
 
-Below is a code sample you can add to your gulp configuration file using the gulp-sass package:
 ```JS
-.pipe(sass({ includePaths: 'node_modules/' }))
+gulp.task('sass', function () {
```

Comment:
Do we need to be even clearer that they will likely already have something like this in their gulp file already, and that the `includePaths` line is the only bit that's new?

Reply:
Please review again, I've added some more info to this section.